bin/list-packages.rb

```diff
@@ -17,9 +17,7 @@ def list_packages(path,glob,rel)
         debs = [];
         sftp.dir.glob(path,glob) { |f| debs << f }
         
-        p=/^[^_-]+/
-        
-        debs.sort {|x,y| y.name.gsub(p,"")<=>x.name.gsub(p,"")}.each do |f|
+        debs.sort {|x,y| Time.at(y.attributes.mtime)<=>Time.at(x.attributes.mtime)}.each do |f|
```

Comment:
Order by date can work with the most of the cases but an order by name, taking into account the version naming, would be better.